### PR TITLE
OAT: Add toggle for default version number

### PR DESCRIPTION
**Pull request by msnyder-msft**
### Summary of changes 🔍 
Adding a toggle to let users select the default version of new models. 
image
image

### Testing 🧪
- use an existing project before the change, add a model, should have version 2 context
- change the toggle to V3, see new models get V3 context
- change toggle to V2, see new models get V2 context

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/OATHeader/internal/ManageOntologyModal/ManageOntologyModal.tsx [hunk 1]
```diff
@@ -17,25 +18,33 @@ import { buildModelId } from '../../../../Models/Services/OatUtils';
...
+    modalSubTitleLinkText: `OAT.Header.ManageOntologyModal.modalSubTitleLinkText`,
+    modalTitleCreate: `OAT.Header.ManageOntologyModal.modalTitleCreate`,
+    modalTitleEdit: `OAT.Header.ManageOntologyModal.modalTitleEdit`,
+    nameLabel: 'OAT.Header.ManageOntologyModal.nameLabel',
+    namePlaceholder: 'OAT.Header.ManageOntologyModal.namePlaceholder',
+    namespaceDescription: 'OAT.Header.ManageOntologyModal.namespaceDescription',
+    namespaceLabel: 'OAT.Header.ManageOntologyModal.namespaceLabel',
+    namespacePlaceholder: 'OAT.Header.ManageOntologyModal.namespacePlaceholder',
+    sampleModel: 'OAT.Header.ManageOntologyModal.sampleModel',
+    toggleLabel: 'OAT.Header.ManageOntologyModal.defaultVersionLabel',
```
Comment (pasanchMSFT) [inline src/Components/OATHeader/internal/ManageOntologyModal/ManageOntologyModal.tsx:45]: nit: radio button not toggle

File: src/Components/OATPropertyEditor/Internal/JSONEditor.tsx [hunk 2]
```diff
@@ -191,7 +191,8 @@ const JSONEditor: React.FC<JSONEditorProps> = (props) => {
                 options={{
                     minimap: {
                         enabled: false
-                    }
+                    },
+                    readOnly: true
```
Comment (pasanchMSFT) [inline src/Components/OATPropertyEditor/Internal/JSONEditor.tsx:195]: Why was this added?

File: src/Models/Constants/OatConstants.tsx [hunk 3]
```diff
@@ -48,15 +48,15 @@ export const getSchemaTypeMenuOptions = (
             }
         };
         switch (x.category) {
+            case 'primitive':
```
Comment (pasanchMSFT) [inline src/Models/Constants/OatConstants.tsx:51]: Any reason why order changed here?

**Review by pasanchMSFT — APPROVED** [review]
LGTM, just have some questions to get more context and a micro-nit

File: src/Models/Constants/OatConstants.tsx (hunk 3 above)
Comment (msnyder-msft) [reply src/Models/Constants/OatConstants.tsx:51]: well i wanted to swap the order of the items. But I realized i needed to do it a bit further down. Fixed that now.

File: src/Components/OATHeader/internal/ManageOntologyModal/ManageOntologyModal.tsx (hunk 1 above)
Comment (msnyder-msft) [reply src/Components/OATHeader/internal/ManageOntologyModal/ManageOntologyModal.tsx:45]: Done

File: src/Components/OATPropertyEditor/Internal/JSONEditor.tsx (hunk 2 above)
Comment (msnyder-msft) [reply src/Components/OATPropertyEditor/Internal/JSONEditor.tsx:195]: It's an unrelated ask since we can't fix it before tomorrow.
